### PR TITLE
Include the page layout in the branding page

### PR DESCRIPTION
**Pull request by chamathns**
### Purpose
Improvement for https://github.com/wso2/identity-apps/pull/5868

### Related Issues
- https://github.com/wso2/product-is/issues/20039

### Related PRs
- Related PR `#1` or (None)

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: features/admin-branding-v1/pages/branding.tsx [hunk 1]
```diff
@@ -32,12 +35,53 @@ type BrandingPageInterface = IdentifiableComponentInterface;
...
+        ["data-componentid"]: componentId
+    } = props;
+
+    const { t } = useTranslation();
 
     return (
-        <BrandingCore />
+        <BrandingPreferenceProvider>
+            <PageLayout
+                pageTitle="Branding"
```
Comment (DonOmalVindula) [inline features/admin-branding-v1/pages/branding.tsx:52]:
```suggestion
                pageTitle={ t("extensions:develop.branding.pageHeader.title") }
```

File: features/admin-branding-v1/components/branding-core.tsx [hunk 2]
```diff
@@ -595,236 +595,192 @@ const BrandingCore: FunctionComponent<BrandingCoreInterface> = (
...
+                    <Message
+                        info
+                        floating
+                        attached="top"
+                        className="preview-disclaimer"
+                        content={
+                            (
+                                <>
+                                    { t("extensions:develop.branding.publishToggle.hint") }
+                                </>
```
Comment (DonOmalVindula) [inline features/admin-branding-v1/components/branding-core.tsx:610]:
```suggestion
                        content={ t("extensions:develop.branding.publishToggle.hint") }
```

File: features/admin-branding-v1/components/branding-core.tsx [hunk 3]
```diff
@@ -595,236 +595,192 @@ const BrandingCore: FunctionComponent<BrandingCoreInterface> = (
...
+                )
+            }
+            {
+                showSubOrgBrandingUpdateAlert
+                    ? (
+                        <Alert onClose={ () => setShowSubOrgBrandingUpdateAlert(false) } severity="warning">
+                            { t("extensions:develop.branding.notifications.update.successWaitingAlert.description",
+                                { tenant: tenantDomain }) }
+                        </Alert>
+                    ) : null
```
Comment (DonOmalVindula) [inline features/admin-branding-v1/components/branding-core.tsx:624]:
```suggestion
                showSubOrgBrandingUpdateAlert
                    && (
                        <Alert onClose={ () => setShowSubOrgBrandingUpdateAlert(false) } severity="warning">
                            { t("extensions:develop.branding.notifications.update.successWaitingAlert.description",
                                { tenant: tenantDomain }) }
                        </Alert>
                    )
```

File: features/admin-branding-v1/components/branding-core.tsx [hunk 4]
```diff
@@ -595,236 +595,192 @@ const BrandingCore: FunctionComponent<BrandingCoreInterface> = (
...
+            }
+            {
+                showSubOrgBrandingDeleteAlert
+                    ? (
+                        <Alert onClose={ () => setShowSubOrgBrandingDeleteAlert(false) } severity="warning">
+                            { t("extensions:develop.branding.notifications.delete.successWaitingAlert.description",
+                                { tenant: tenantDomain }) }
+                        </Alert>
+                    ) : null
```
Comment (DonOmalVindula) [inline features/admin-branding-v1/components/branding-core.tsx:633]:
```suggestion
                showSubOrgBrandingDeleteAlert
                    && (
                        <Alert onClose={ () => setShowSubOrgBrandingDeleteAlert(false) } severity="warning">
                            { t("extensions:develop.branding.notifications.delete.successWaitingAlert.description",
                                { tenant: tenantDomain }) }
                        </Alert>
                    )
```

**Review by brionmario — APPROVED** [review]

File: features/admin-branding-v1/pages/branding.tsx (hunk 1 above)
Comment (chamathns) [reply features/admin-branding-v1/pages/branding.tsx]: Addressed with https://github.com/wso2/identity-apps/pull/5871/commits/3ee321be7f3fc5b52970870b5e6d80d3200ac10b

File: features/admin-branding-v1/components/branding-core.tsx (hunk 2 above)
Comment (chamathns) [reply features/admin-branding-v1/components/branding-core.tsx]: Added with https://github.com/wso2/identity-apps/pull/5871/commits/3ee321be7f3fc5b52970870b5e6d80d3200ac10b

File: features/admin-branding-v1/components/branding-core.tsx (hunk 3 above)
Comment (chamathns) [reply features/admin-branding-v1/components/branding-core.tsx]: Added with https://github.com/wso2/identity-apps/pull/5871/commits/3ee321be7f3fc5b52970870b5e6d80d3200ac10b

File: features/admin-branding-v1/components/branding-core.tsx (hunk 4 above)
Comment (chamathns) [reply features/admin-branding-v1/components/branding-core.tsx]: Added with https://github.com/wso2/identity-apps/pull/5871/commits/3ee321be7f3fc5b52970870b5e6d80d3200ac10b

**Review by DonOmalVindula — APPROVED** [review]

**Pull request merged**